Include/dpsim/MNASolver.h

```diff
@@ -93,6 +91,15 @@ namespace DPsim {
 		///
 		std::vector< CPS::Attribute<Matrix>::Ptr > mLeftVectorHarmAttributes;
 
+		// #### MNA specific attributes related to system recomputation
+		/// Number of system matrix recomputations
+		Int mNumRecomputations;
```

Comment:
@dinkelbachjan Is this aggregating the number of recomputations of all time steps?

Reply:
Yes indeed. It was introduced to obtain the overall number of recomputations (refactorisations) performed for a single simulation.